app/jobs/super_good/solidus_taxjar/replace_transaction_job.rb

```diff
@@ -6,14 +6,21 @@ class ReplaceTransactionJob < ApplicationJob
       queue_as { SuperGood::SolidusTaxjar.job_queue }
 
       def perform(order)
-        refund_transaction, order_transaction = SuperGood::SolidusTaxjar.reporting.refund_and_create_new_transaction(order)
-
-        SuperGood::SolidusTaxjar::TransactionSyncLog.create!(
-          order: order,
-          refund_transaction: refund_transaction,
-          order_transaction: order_transaction,
-          status: :success
-        )
+        begin
+          refund_transaction, order_transaction = SuperGood::SolidusTaxjar.reporting.refund_and_create_new_transaction(order)
+          SuperGood::SolidusTaxjar::TransactionSyncLog.create!(
+            order: order,
+            refund_transaction: refund_transaction,
+            order_transaction: order_transaction,
+            status: :success
+          )
+        rescue Taxjar::Error => exception
```

Comment:
I think it could be worth adding a code comment that explains that this is so that we create an admin-facing sync error.